### PR TITLE
Create git commands

### PR DESCRIPTION
**Pull request by oliva90**
this is homework number 2

### CODE REVIEW

**Review by showmic09 — COMMENTED** [review]
@oliva90 Please change the directory name. You have `OlivaHW2` as the folder name. Moreover, this is not the correct homework. I asked for shell commands like: ls, cd, pwd.

**Review by showmic09 — CHANGES_REQUESTED** [review]
Also please make them as a bullet list.

File: olivaHW2/HW2/git commands
Comment (oliva90) [inline olivaHW2/HW2/git commands:1]: how do I change the name

**Review by showmic09 — APPROVED** [review]
LGTM

**Pull request merged**